app/src/components/governance/LiProposal.vue

```diff
@@ -18,15 +18,15 @@
         </h3>
       </div>
       <span v-if="proposal.creationTime" class="time">{{
-        proposal.creationTime | moment
```

Comment:
This should be done in another PR. One only dedicated to dates like the one Mario is doing